index.d.ts

```diff
@@ -456,7 +456,10 @@ export interface Player {
   gamemode: number
   ping: number
   entity: Entity
-}
```

Comment:
Uhm... 
![image](https://user-images.githubusercontent.com/53813973/195616118-5c2e9371-9d3d-43df-b045-65bcf4ce91e9.png)
And I wondered why my project won't compile...